File: 프로그래머스/unrated/181894. 2의 영역/2의 영역.py
def solution(arr):
    start = -1
    end = -1
    
    for i in range(len(arr)):
        if arr[i] == 2 and start == -1:
            start = i
            end = i
        elif arr[i] == 2:
            end = i
    
    return [-1] if start == -1 else arr[start:end + 1]
            

File: 프로그래머스/unrated/181894. 2의 영역/2의 영역.js
function solution(arr) {
    let start = -1
    let end = -1
    
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] === 2 && start === -1) {
            start = i
            end = i
        } else if (arr[i] === 2) {
            end = i
        }
    }
    
    return start === -1 ? [-1] : arr.slice(start, end + 1)
}